docs/templates/preprocessing/image.md

```diff
@@ -30,6 +30,10 @@ Generate batches of tensor image data with real-time data augmentation. The data
     - __shear_range__: Float. Shear Intensity (Shear angle in counter-clockwise direction as radians)
     - __horizontal_flip__: Boolean. Randomly flip inputs horizontally.
     - __vertical_flip__: Boolean. Randomly flip inputs vertically.
+    - __output_size__: Tuple. Resize images (tuple of height and width according to dim_ordering)
+    - __position__: One if {"random", "center", "top left"}.
+        defines where to position a image in the resized one
+    - __background__: Float. Background color to be used for resized image
```

Comment:
Okay, what I try to accomplish with this that one can input images with different sizes and that the generator outputs them with a fixed size. The images are neither resized or crop, they are copied into the output, which is created using ```np.full(...```, ```output_size``` and ```background```. Depending on what was done with the images, e.g. graysacle, [0,1], it should be a float. The ```position``` determents how it is copied. I have only used/implemented these there, that's the only reason why there aren't others. I particularly like the random position since it seems that the net learns some positional invariance. I give it another try to explain it in the docu.